.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+<!--
+This issue tracker for bug reports and feature requests only.
```

Comment:
Shouldn't feature request be created in rfcs repo actually?

Reply:
I would consider RFC repo feature request more sacrosanct than one opened on the GitHub issue. RFC requires a well thought out and researched proposal. Before one does that it would be nice to start a discussion here and check the interest of other users and contributors.